docs/test-plans/005-Block-Sync/64-128sq-plan.md

```diff
@@ -0,0 +1,86 @@
+# Block Synchronization for 64/128/256 Square Size Block
+
+We need to run experiments to know how fast will it be for the full node(s) to sync blocks from bridge nodes and from amongst themselves, as the blocks are being produced for
+both IPLD and Shrex getters.
+
+In this scenario we are only focusing on syncing, thus ignoring any other scenario (_such as attacks of any kind_)
+
+## Pre-Requisite
+
+| Block Square Size<br />`X` |            Bridge Nodes amount <br />`Y`             |  Full Nodes Amount <br />`Z` | Peer Limit <br />`W` |
+| :---------------------------: | :-----------------------------------------: | :---------------------------: | :---------------------------: |
+|              64               | 3 | 32 | 3 |  
+|              128               |  3  | 32 | 3 | 
+
+And For Historical Scenarios
+
+| Block Square Size<br />`X` |            Bridge Nodes amount <br />`Y`             |  Full Nodes Amount <br />`Z` | Historical Full Nodes <br />`H` | Peer Limit <br />`W` |
+| :---------------------------: | :-----------------------------------------: | :---------------------------: | :---------------------------: | :---------------------------: |
+|              64               | 3 | 12 | 12 |   3 |  
+|              128               |  3  | 12 | 12 |  3 |
+
+## HW Resources setup for Block Square Size ==  64|128|256
+
+We already know that we can count on 172 cores CPU and 700 GB of RAM to allocate for testable nodes
+Hence, our network setup/HW allocation will look like this:
+
+1. 1 Validators with each having 1 CPU and 4 Gb
+2. 3 Bridge Nodes with each having 1 CPU and 8 Gb
+3. 32 Full Nodes with each having 1 CPU and 8 Gb
+
+## Network Bandwidth:
+
+- Validator/Bridge/Full: 256Mib -- 50ms
+
+## Network Topology
+
+### All cases
+For all cases, the singular validator will be connected to all three bridges.
+
+### Case 1 - Syncing Latest:
+
+Z Full nodes all connected to some or all of the existing Y bridges, and connected amongst themselves (_peerLimit is set to W_) such that full nodes are syncing latest blocks as they are being produced.
```

Comment:
```suggestion
Z Full nodes all connected to all of the existing Y bridges, and connected amongst themselves (_peerLimit is set to W_) such that full nodes are syncing latest blocks as they are being produced.
```

Reply:
Or please be more precise where the `some` is defined in test data

Reply:
some means exactly one, as per the logic that picks bridge nodes for fulls or lights, correcting...